crates/wasi-nn/tests/all.rs

```diff
@@ -90,3 +94,9 @@ fn nn_image_classification() {
 fn nn_image_classification_named() {
     run(NN_IMAGE_CLASSIFICATION_NAMED, true).unwrap()
 }
+
+#[cfg_attr(not(all(target_arch = "x86_64", target_os = "windows")), ignore)]
+#[test]
+fn nn_image_classification_winml() {
+    run(NN_IMAGE_CLASSIFICATION_WINML, true).unwrap()
```

Comment:
You will want to double-check that this test is actually run. Right now in Wasmtime's CI, only a subset of jobs run to preserve runner time. The Windows "test" job does not run by default but if you add the string `prtest:full` in any commit, then all jobs will run. Then we can check the logs for the Windows job to ensure this test actually ran.

Reply:
With prtest:full, this test failed on Windows Server. I guess GitHub Windows Server runner doesn't have Desktop Experience enabled.

Reply:
I'm afraid we need to disable this test at this time. GitHub Actions Windows Server image doesn't have Desktop Experience available, so it cannot be installed.

List of installed or available features:
https://github.com/jianjunz/wasmtime/actions/runs/7752997447/job/21143496106#step:9:1

Reply:
Yeah, that's unfortunate. Why don't we add a function like `check_openvino_is_installed` but for WinML: `check_winml_is_available`. There's probably a way to check with the Windows API if ML system calls are possible and that function could return false if they are not.

That extra runtime check would be in addition to the `#[cfg_attr(not(feature = "winml"), ignore)]`. That can stay there for the case where the `winml` feature isn't even enabled.